front/cypress/e2e/routes/scene/Scene.cy.js

```diff
@@ -28,13 +29,14 @@ describe('Scene view', () => {
       };
       cy.createDevice(device, 'mqtt');
     });
+     */
```

Comment:
Is it normal that it's commented?

Reply:
Not needed now. I have moved this device creation into the good test to avoid some issue